Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.11.6-alpine3.18
```

Comment:
The base image `python:3.11.6-alpine3.18` is a good choice for a thin image, which aligns with the checklist requirement to keep the image as thin as possible.